Documentation/Input/Pointers.md

```diff
@@ -245,6 +245,32 @@ The [PointerResultExample scene](https://github.com/microsoft/MixedRealityToolki
 
 <img src="../../Documentation/Images/Input/PointerResultExample.png" style="max-width:100%;">
 
+### Turn pointers on and off
+To turn pointers on and off, set the [`PointerBehavior`]() for a given pointer type via [`PointerUtils`](). See [How to turn pointers on and off in MRTK using PointerBehavior]() for more information.
```

Comment:
I think that this is missing links in PointerBehavior and PointerUtils and [How to turn pointers on and off in MRTK using PointerBehavior